shared/js/client_api/properties.js

```diff
@@ -1,64 +0,0 @@
-function sharedProperties(channel_id)
```

Comment:
currently unused by initapp. eventually we'll have a shared clientapi used by initApp and apps